tests/rundoctest.py

```diff
@@ -8,12 +8,18 @@
 path = os.path.dirname(__file__)
 if path:
     os.chdir(path)
+os.chdir('..')
```

Comment:
`os.chdir` does not change the python module search path, and the search path for a script run with `python script.py` does not contain `.`.

The incriminated line would rather be the following one, that adds the cypari2 directory to `sys.path`. But it only adds that to the end of `sys.path`, so it does not take precedence over the **installed** cypari2.